integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -3258,37 +3254,36 @@ msgstr "Rollen"
 
 #: cms/models/users/user.py:57
 msgid "The regions to which the user has access"
-msgstr "Die Regionen, auf die der Benutzer Zugriff hat"
+msgstr "Die Regionen, auf die zugegriffen werden kann"
 
 #: cms/models/users/user.py:67
 msgid ""
 "This allows the user to edit and publish all pages for which the "
 "organisation is registered as the responsible organisation"
 msgstr ""
-"Dies erlaubt dem Benutzer, alle Seiten zu bearbeiten und zu veröffentlichen, "
-"für die die Organisation als zuständige Organisation eingetragen ist"
+"Dies erteilt die Berechtigung, alle Seiten zu bearbeiten und zu "
+"veröffentlichen, für die die Organisation als zuständige Organisation "
+"eingetragen ist"
 
 #: cms/models/users/user.py:72
 msgid "last chat visit date"
 msgstr "Datum des letzten Chat-Besuchs"
 
 #: cms/models/users/user.py:73
 msgid "The date and time when the user did read the chat the last time"
-msgstr ""
-"Das Datum und die Uhrzeit, wann der Benutzer den Chat das letzte Mal gelesen "
-"hat"
+msgstr "Das Datum und die Uhrzeit, wann der Chat das letzte Mal gelesen wurde"
 
 #: cms/models/users/user.py:77
 msgid "experienced user"
-msgstr "Erfahrener Anwender"
+msgstr "Erfahrene*r Anwender*in"
 
 #: cms/models/users/user.py:79
 msgid ""
 "Enable this option to display additional features like XLIFF import/export, "
 "page filtering, mirrored pages, page-based permissions, do-not-translate-tag "
 "and status information for broken links"
 msgstr ""
-"Aktiviert den Expertenmodus und ermöglicht zusätzliche Features wie XLIFF-"
+"Aktiviert den Expertisen-Modus und ermöglicht zusätzliche Features wie XLIFF-"
```

Comment:
Below, it is called "Expertise-Modus" and here it is "Expertisen-Modus"... I don't really have a preference, but probably we should use the same wording in all occasions.

Reply:
(Rejected) We landed on Expert:innenmodus